docs/guide/apm-release-notes.asciidoc

```diff
@@ -26,6 +27,80 @@ For a full list of changes, see the
 
 // end::notable-v8-highlights[]
 
+[[release-highlights-7.6.0]]
+=== APM version 7.6.0
+
+// tag::notable-v76-highlights[]
+Jaeger integration::
+
+Elastic APM now integrates with https://www.jaegertracing.io/[Jaeger], an open-source, distributed tracing system.
+This integration allows users with an existing Jaeger setup to switch from the default Jaeger backend,
+to ingesting data with Elastic's APM Server, storing data in {es}, and visualizing traces in the APM app.
+Best of all, this can be done without any instrumentation changes in your application code.
++
+See {apm-server-ref-v}/jaeger.html[Jaeger integration] for more information.
+
+API Key authentication::
+
+You can now secure the communication between APM Agents and APM Server with
+{apm-server-ref-v}/api-key.html[API keys].
+API keys are easy to create and manage, and provide finer authorization rules than secret tokens. 
+
+SIEM integration::
+
+HTTP data collected with Elastic APM is now available in the SIEM app,
+enabling you to hunt for security threats with your APM data.
++
+See {siem-guide}/siem-ui-overview.html[SIEM UI] for information on using the SIEM app.
```

Comment:
@benskelker How's this look? Any recommendations?

Reply:
@bmorelli25 
Hey, sounds good.
Good that you pinged me, I should add a sentence about APM being a default data source in the SIEM app - thanks.

Reply:
Awesome!